ASWF/meetings/tsc/2020-08-31.md

```diff
@@ -0,0 +1,133 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the OpenColorIO Project. -->
+
+August 31, 2020
+
+Host: Michael Dolan
+
+Attendees:
+  * [ ] Mark Boorer (_TSC_) - Industrial Light & Magic
+  * [X] Mei Chu (_TSC_) - Sony Pictures Imageworks
+  * [ ] Sean Cooper (_TSC ACES TAC Rep_) - DNEG
+  * [X] Michael Dolan (_TSC Chair_) - Epic Games
+  * [X] Patrick Hodoul (_TSC_) - Autodesk
+  * [X] John Mertic - Academy Software Foundation / Linux Foundation
+  * [X] Carol Payne (_TSC_) - Netflix
+  * [ ] Mark Titchener (_TSC_) - Foundry
+  * [ ] Carl Rand (_TSC_) - Weta Digital
+  * [X] Doug Walker (_TSC Chief Architect_) - Autodesk
+  * [ ] Kevin Wheatley (_TSC_) - Framestore
+  * [X] Bernard Lefebvre - Autodesk
+  * Daniele Siragusano - FilmLight
+  * Joseph Goldstone - ARRI
+  * Anandhaiyappan M
+
+Apologies:
+  * Mark Titchener
+
+# **OCIO TSC Meeting Notes**
+
+* Issue #1124 discussion and resolution
+    - London holiday today so we don't have quorum. We can discuss solutions to 
+      present at the next TSC meeting where all members can provide additional 
+      input and decide on a resolution together.
+    - Any additional thoughts to share for the issue discussion?
+    - Doug: Was discouraging to read after all the work, but the nature of open 
```

Comment:
@doug-walker : I have said it many times but I'm happy to repeat it again: thank you for all the work your team has accomplished, OCIO would simply not be where it is without your work. I understand that the way I have written #1124 can be perceived as being blunt, so while I maintain my position, I also send my sincerest apologies to yourself, @hodoulp and @BernardLefebvre if you were offended because it was certainly not the intent.

Reply:
No worries @KelSolaar .  But I do want to clarify this bullet point.  What I was trying to say was that we had felt that we were actually doing a pretty good job at communicating and being transparent, so to have someone as knowledgeable about open source as yourself say that we're not doing a good job was discouraging.

As I benchmark OCIO against our peer ASWF projects, I still feel that we have been doing at least ok.  It's true that OpenCue and OTIO are making more effective use of GitHub Projects boards than we are.  However, I'm not aware of any of the other projects making regular 90 minute power-point presentations to get feedback on new features, as we have been doing.  And I feel that generally we actually have been doing a good job at creating GitHub Issues for new features.  The example you cite was one of the rare exceptions over the three years.  But let's not dwell on that, the main point is that we are here to learn and absorb best-practices, where feasible.

For the record, as long as Michael agrees this is consistent with my actual remarks, I'd like to amend the bullet point to be:
- Doug: Transparency and communication was actually something we felt we had been doing a good job on, so it was discouraging to hear that we still have more work to do on that front.  The distributed nature of open source development does make it difficult for people to track, especially if they cannot attend all of the live discussions.  If we can make additional improvements in this area, I think they would be appreciated.

Reply:
Hi Doug,

> so to have someone as knowledgeable about open source as yourself say that we're not doing a good job was discouraging

There is a misunderstanding here: I think I framed the problem being #1102 mostly. It is a large outlier PR that came very much unannounced, made myself and other key people from the industry (I wish they show their heads :) ) uncomfortable. Should have it been a small feature PR without any complex ramifications for the future of the project, I would not have said anything and maybe we would not be discussing at all. In turn, you would have spent a better weekend and so I would have. It is certainly not easy to write and make those things public and by doing so I'm exposing myself to criticism which I reckon is an activity that nobody particularly enjoys.

> And I feel that generally we actually have been doing a good job at creating GitHub Issues for new features.

I completely agree with that and I don't think I have said the opposite or at least intended to say the opposite. Quoting myself and this is really the gist of it, with #1102 being the trigger:

> Many transparent pieces scattered everywhere does not necessarily directly correlate with a process that is perceived as fully transparent.

This is the bit I would like us to solve so that people are always comfortable with the direction the project is heading to.

Reply:
Thanks for the clarification @KelSolaar but I did realize that your comments were informed by private conversations you had.  Thank you for being brave and willing to step up and take the arrows for your friends.  But to be clear, no one at the TSC today was criticizing you for doing this (so there were no arrows flying!).  When I said it was "discouraging", I did not mean that it was discouraging that you would make the comment.  It was discouraging because I thought we were doing better than we actually are, apparently.

I think we did understand your point about things being scattered and too difficult to find.  I think Michael's proposals that we agreed to implement today are a step towards addressing that very point.  We will look forward to getting further feedback from you to see if that has solved the issue.

Reply:
Just makes me wish there was a physical colour bar, talking IRL around a pint of beer would make all that much easy :)

Reply:
Statement revised.

Reply:
@KelSolaar , I look forward to meeting you at the x-bar, y-bar, or z-bar when this is all over for a pint!
![image](https://user-images.githubusercontent.com/43830961/91887349-ace22380-ec58-11ea-91cd-1a6171a6b350.png)